azure-devops/azext_devops/dev/pipelines/_help.py

```diff
@@ -48,3 +48,15 @@ def load_pipelines_help():
     short-summary: Manage release definitions.
     long-summary:
     """
+
+    helps['pipelines runs'] = """
+    type: group
+    short-summary: Manage pipeline runs.
+    long-summary:
+    """
+
+    helps['pipelines runs artifact'] = """
+    type: group
+    short-summary: Download, upload and list run artifacts.
```

Comment:
Short summary: "Manage pipelines artifacts" should be enough so we do not have to revisit the help when we add more commands under the artifact subgroup and consistent with other summary help texts.